### PR TITLE
Add conversion to parquet as part of data generation

### PR DESCRIPTION
**Pull request by nikhil96sher**
This PR also adds the logic to convert the generated tbl files to parquet format as part of the data generation script.

### CODE REVIEW

File: scripts/data-gen.sh [hunk 1]
```diff
@@ -2,12 +2,15 @@ if [ "$#" -ne 3 ]; then
...
 scale=$1
 partition=$2
-BASE_DIR=$3
-export DSS_PATH=$BASE_DIR/resources/tpc-h/data/scale=$scale/partition=$partition/tbl
+output_dir=$3
+export DSS_PATH=$output_dir/scale=$scale/partition=$partition/tbl
 mkdir -p $DSS_PATH
 echo $DSS_PATH
+
 cd ../tpch-kit/dbgen/
```
Comment (mIXs222) [inline scripts/data-gen.sh:14]: You could do `pushd` here

File: scripts/data-gen.sh [hunk 2]
```diff
@@ -20,3 +23,7 @@ else
 			./dbgen -f -s $scale -S $chunk -C $partition -v
 		done
 fi
+
+echo "Output Directory: $DSS_PATH"
+cd $cwd
```
Comment (mIXs222) [inline scripts/data-gen.sh:28]: Then `popd` here

**Review by mIXs222 — APPROVED** [review]
LGTM

File: scripts/data-gen.sh (hunk 2 above)
Comment (nikhil96sher) [reply scripts/data-gen.sh]: Thanks! Changed it to `pushd` and `popd`.

**Pull request merged**